en/components/general/cli/getting-started-with-cli.md

```diff
@@ -14,6 +14,12 @@ To get started install [Ignite UI CLI](https://github.com/IgniteUI/igniteui-cli)
 npm install -g igniteui-cli 
 ```
 
+If you are using `yarn` package manager:
+
+```cmd
+yarn global add igniteui-cli
```

Comment:
@jackofdiamond5 @zdrawku 
This can be a bit misleading, since yarn will happily the package, but afterwards once you call `ig new` the Ignite UI CLI will still default to npm and you'll get a `package-lock.json` instead of `yarn.lock` which might not be what the user expects. 

We have some stub logic for diff package managers, but that's about it, technically the new command should have a param for that like https://angular.io/cli/new has `--package-manager` or something similar.

So not sure we should be instructing users to do this via yarn for now

Reply:
As Damyan said there was an idea at some point to add other package managers:
![image](https://user-images.githubusercontent.com/11193764/177503353-1c229a4d-b7b5-4dcc-af61-78b61d5749d2.png)

If there is a customer request for this and people from webinars are asking for this, maybe adding yarn support is a good candidate for some of our future plannings.

Adding @Lipata as well.

IMHO, if we guesstimate a rough scope for this feature we can actually decide whether to plan it for implementation or not.